### ISSUE TITLE
Accept header on GET request

### ISSUE THREAD

**Issue by gsverma** [miLibris/flask-rest-jsonapi#75]
Does not JSONAPI 1.0 says, GET requests be made with Accept header "application/vnd.api+json"; But, I am able to make GET requests without any headers. 

Also, is response body in GET according to JSONAPI specs? Our app broke as it was not able to parse response body (It was working fine with restless before migrate). 
Or, do I need to put in some options in API library to change mime type to application/vnd.api+json? I thought that to be default.

**Comment by gsverma** [comment]
@akira-dev @Natureshadow please let me know of possible cause. We are short of time on this one. thanks.

**Comment by hellupline** [comment]
@gsverma can you give any details how it the GET response differs from yours restless before migrate ?
AFAIK, there should not be a problem if the client didn't set the `Accept` header

dont know about the ajax lib you're using, but `curl` and `httpie` set `Accept: */*`, 

also, `flask-rest-jsonapi` sets `Content-Type: application/json`, which should be enough for js `jquery` and python `requests`

**Comment by gsverma** [comment]
@hellupline  JSONAPI specs require the content type to be set as "Content-Type: application/vnd.api+json" and not Content-Type: application/json.  (http://jsonapi.org/format/ )
Same goes for GET requests to API endpoints, GET requests should be made be Accept header set as application/vnd.api+json. 

Anyways, Content-Type: application/vnd.api+json is still there in flask-rest-jsonapi for POST and PATCH requests; But, Accept header is missing in GET. It simply does not return any error if Accept header is not set.  

Although, above behavior with Accept header is only on this repo's master branch (v0.13.2). Luckily, develop branch (v0.12.5) supports JSONAPI specs strictly. So, using that one now. 

We just like following all specs a bit seriously, that's it. So, was the reason for moving to flask-rest-jsonapi; Flask-Restless was just placeholder to get our front end guys get started on app. We were anyways planning to move to Flask-Restful for its flexibility, flask blueprint support and so much more. Only their request parser wasn't jsonapi compliant. Hence, I was experimenting around Restful to integrate it with marshmallow_jsonapi; then I came across flask-rest-jsonapi.  It was only recently, when we were freezing restless to staging and started seriously on flask-rest-jsonapi (using its master), I noticed it allows GET requests with missing headers too. Hope this gets resolved soon. And, how come develop has lower version number and supports a desired feature, which is missing in a later version? @akira-dev
